app/ChordNode.js

```diff
@@ -428,7 +432,12 @@ class ChordNode {
       this.predecessor = this.encapsulateSelf();
     }
 
-    await this.migrateKeys();
+    try {
+      console.log("join: calling migrateKys");
```

Comment:
Can you please put this behind the DEBUGGING_LOCAL thing the other informational messages are behind? I'm trying to make sure all of our debugging logic hooks off that so we can improve how it works in the future.

Reply:
Done